sphinx/source/start_network.rst

```diff
@@ -44,37 +44,63 @@ To join an existing network, other nodes should be started with the ``join`` opt
     --node-cert-file /path/to/node_certificate --quote-file /path/to/quote
     join --network-cert-file /path/to/existing/network_certificate --target-rpc-address target_rpc_ip:target_rpc_port
 
-The node takes the certificate of the existing network to join via ``--network-cert-file`` and initiates an enclave-to-enclave TLS connection to an existing node of the network as specified by ``--target-rpc-address``. Once the join protocol [#remote_attestation]_ completes, the joining node becomes part of the network as a backup (see :ref:`Ledger replication` for more details on consensus protocols).
+The joining node takes the certificate of the existing network to join via ``--network-cert-file`` and initiates an enclave-to-enclave TLS connection to an existing node of the network as specified by ``--target-rpc-address``. Once the join protocol [#remote_attestation]_ completes, the joining node becomes part of the network as a backup (see :ref:`Ledger replication` for more details on consensus protocols).
 
 .. note:: When starting up the network or when joining an existing network, the network secrets required to decrypt the ledger are sealed and written to a file so that the network can later be recovered. See :ref:`Catastrophic Recovery` for more details on how to recover a crashed network.
 
-Opening a network
-~~~~~~~~~~~~~~~~~
+Opening a network to users
+--------------------------
 
-Once a CCF network is successfully started and an acceptable number of nodes have joined, members should vote to open the network to users. Users will then be able to execute transactions on the business logic defined by the enclave file (``--enclave-file`` option to ``cchost``).
+Adding users
+~~~~~~~~~~~~
 
-For example, the first member may decide to make a proposal to open the network:
+Once a CCF network is successfully started and an acceptable number of nodes have joined, members should vote to open the network to users. First, the certificates of trusted users should be registered in CCF via the member governance interface. For example, the first member may decide to make a proposal to add a new user (here, ``user_cert`` is the PEM certificate of the user -- see :ref:`Cryptography` for a list of supported algorithms):
```

Comment:
Should probably add a note making it clear that users can also be added after the network is opened because this section implies that they need to be added before

Reply:
The note is already in the `Summary Diagram` section:

```
.. note:: After the network is open to users, members can still issue governance transactions to CCF (for example, adding new users or additional members to the consortium). See :ref:`Governance` for more information about member governance.
```